assets/js/components/dashboard-sharing/ModuleRecoveryAlert/index.js

```diff
@@ -19,21 +19,31 @@
 /**
  * WordPress dependencies
  */
-import { Fragment, useCallback, useEffect, useState } from '@wordpress/element';
+import {
+	createInterpolateElement,
+	Fragment,
+	useCallback,
+	useEffect,
+	useState,
+} from '@wordpress/element';
 import { sprintf, __ } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { useSelect, useDispatch } from 'googlesitekit-data';
-import { Checkbox, ProgressBar } from 'googlesitekit-components';
+import { Checkbox } from 'googlesitekit-components';
 import { CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
 import { CORE_MODULES } from '../../../googlesitekit/modules/datastore/constants';
 import { DAY_IN_SECONDS } from '../../../util';
-import BannerNotification from '../../notifications/BannerNotification';
 import Errors from './Errors';
+import SimpleNotification from '../../../googlesitekit/notifications/components/layout/SimpleNotification';
+import Link from '../../Link';
+import CTALink from '../../../googlesitekit/notifications/components/common/CTALink';
+import CTALinkSubtle from '../../../googlesitekit/notifications/components/common/CTALinkSubtle';
+import { CORE_NOTIFICATIONS } from '../../../googlesitekit/notifications/datastore/constants';
 
-export default function ModuleRecoveryAlert() {
+export default function ModuleRecoveryAlert( { id, Notification } ) {
```

Comment:
Most components are simplified during the transition to the new API, however this one has grown substantially, at least in terms of total lines. Perhaps this is mostly due to my previous comment about using `Description` components instead of defining inline here, but even beyond that, it seems like there is room for some additional refactoring to simplify things further.

Reply:
I've much reduced the size, although it is larger as the props which were once on the BannerNotification component are now duplicated on CTA/Dismiss components.

I've also added additional functionality to disable the Recover button when no modules are selected.